assets/src/components/URLInput/URLInput.js

```diff
@@ -0,0 +1,50 @@
+import {Component} from '@wordpress/element';
+import {TextControl} from '@wordpress/components';
+
+export class URLInput extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+
+    const {__} = wp.i18n;
+    const { label, placeholder, value, onChange, disabled, help } = this.props
+
+    let errorMessage = "";
+
+
+    const isCustomLinkValid = () => {
+      if ( !this.props.value ) {
+        return true;
+      }
+
+      try {
+        const url = new URL(this.props.value);
+        if ( url.protocol !== 'https:') {
+          errorMessage = 'The URL must start with "HTTPS://"';
```

Comment:
Probably would be more recognizable to the user to have `https` in lowercase?

Reply:
possibly. However the spec of the ticket asked for it to be upper case. It is possibly to highlight it.